test/test/unit/test_link.py

```diff
@@ -287,6 +287,15 @@ def test_generic_error_raised(self, inlet1):
         # with self.assertRaisesRegex(TypeError, databay.link._ITERABLE_EXCEPTION):
         self.assertRaises(DummyException, link.transfer)
 
+    @patch(fqname(Inlet),spec=Inlet)
+    def test_integer_to_timedelta(self, inlet1):
+        link = Link([inlet1], [], 1, name='test_integer_interval_coerced')
+        self.assertEquals(link._interval, timedelta(seconds=1))
+    
+    @patch(fqname(Inlet),spec=Inlet)
```

Comment:
It's fantastic you're trying out mocks already! I only feel that in these particular two test cases we could get away without mocking? Link can be created without inlets or outlets passed, and since we're testing a functionality that is independent from either, the test would be more specific if we decouple it from these two. So my suggestion would be to skip the mock and call `Link([], [], 1, ...`

Reply:
Yep you're definitely right here, I think I realised post doing the mock I could leave the Link's inlets and outlets empty. Will put in a change for that.

Reply:
Nice one 👏